src/main/java/frc/team2412/robot/subsystem/ClimbSubsystem.java

```diff
@@ -119,7 +119,7 @@ public ClimbSubsystem(WPI_TalonFX climbFixedMotor, WPI_TalonFX climbDynamicMotor
 
         climbFixedMotor.config_kP(PID_SLOT_0, P);
 
-        climbDynamicMotor.configAllSettings(motorConfig);
+        // climbDynamicMotor.configAllSettings(motorConfig);
```

Comment:
Instead, just check if it's non-null so you don't have to keep touching this code as the climb arm is added or removed.

Reply:
Ditto for elsewhere below & where the solenoid is used.